<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgAudioBase</class>
 <widget class="QWidget" name="DlgAudioBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>467</width>
    <height>437</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Audio settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox1_audio">
     <property name="mouseTracking">
      <bool>false</bool>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QFormLayout" name="formLayout_3">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label_interface">
        <property name="text">
         <string>Sound manager</string>
        </property>
        <property name="buddy">
         <cstring>m_pManager</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="KComboBox" name="m_pManager"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QStackedWidget" name="stackedWidget_interfaceSpecificSettings">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page1_alsa">
      <layout class="QVBoxLayout" name="verticalLayout_20">
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="QGroupBox" name="groupBox_alsa">
         <property name="title">
          <string>ALSA Settings</string>
         </property>
         <layout class="QFormLayout" name="formLayout_4">
          <property name="fieldGrowthPolicy">
           <enum>QFormLayout::ExpandingFieldsGrow</enum>
          </property>
          <property name="verticalSpacing">
           <number>5</number>
          </property>
          <property name="leftMargin">
           <number>9</number>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="label1_alsaPugin">
            <property name="text">
             <string>ALSA plugin</string>
            </property>
            <property name="buddy">
             <cstring>box_alsaPlugin</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="KComboBox" name="box_alsaPlugin">
            <property name="sizeAdjustPolicy">
             <enum>QComboBox::AdjustToContents</enum>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label2_in">
            <property name="text">
             <string>Input</string>
            </property>
            <property name="buddy">
             <cstring>alsaInputDevice</cstring>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="KComboBox" name="alsaInputDevice">
            <property name="sizeAdjustPolicy">
             <enum>QComboBox::AdjustToContents</enum>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label3_out">
            <property name="text">
             <string>Output</string>
            </property>
            <property name="buddy">
             <cstring>alsaOutputDevice</cstring>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="KComboBox" name="alsaOutputDevice">
            <property name="sizeAdjustPolicy">
             <enum>QComboBox::AdjustToContents</enum>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="alsaRingtoneDeviceL">
            <property name="text">
             <string>Ringtone</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="KComboBox" name="alsaRingtoneDevice"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page2_pulseAudio">
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <widget class="QGroupBox" name="groupBox_pulseAudio">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>PulseAudio Settings</string>
         </property>
         <layout class="QFormLayout" name="formLayout_11">
          <property name="fieldGrowthPolicy">
           <enum>QFormLayout::ExpandingFieldsGrow</enum>
          </property>
          <item row="0" column="0">
           <widget class="QCheckBox" name="kcfg_pulseAudioVolumeAlter">
            <property name="text">
             <string>Mute other applications during a call</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox1_recordGeneral">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Recording</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="1">
       <widget class="KUrlRequester" name="KUrlRequester_destinationFolder">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="clickMessage">
         <string>Destination folder</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QCheckBox" name="m_pAlwaysRecordCK">
        <property name="text">
         <string>Always record calls</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Audio settings</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QCheckBox" name="m_pSuppressNoise">
        <property name="text">
         <string>Enable noise reduction</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="kcfg_enableRoomTone">
        <property name="text">
         <string>Enable dial tone</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="m_pCPlayDTMFCk">
        <property name="text">
         <string>Mute DTMF</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>KUrlRequester</class>
   <extends>QFrame</extends>
   <header>kurlrequester.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
